compiler/backend/README.md

```diff
@@ -7,6 +7,9 @@ backend and associated proofs.
 [arm6](arm6):
 This directory contains the ARMv6-specific part of the compiler backend.
 
+[arm7](arm7):
+This directory contains the ARMv6-specific part of the compiler backend.
```

Comment:
This comment looks incorrect. (Presumably because the corresponding file comment it was generated from is wrong.)